mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -31,13 +30,17 @@ import org.apache.spark.sql.DataFrame
  */
 @Since("1.2.0")
 class RegressionMetrics @Since("1.2.0") (
-    predictionAndObservations: RDD[(Double, Double)]) extends Logging {
+    predictionAndObservations: RDD[(Double, Double)], isUnbiased: Boolean = true)
```

Comment:
I will double check this. I've seen this being called biased. The problem is that this special case is rarely discussed in literature so the documentation is scarce.